docs/getting_started/basic/zero-shot-prediction/zero-shot-segmentation.ipynb

```diff
@@ -0,0 +1,1402 @@
+{
```

Comment:
<p>...instantiating the Operator that will use the Florence2 model to generate captions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thesteve0/v51-gs-draft/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>